cms/models/contentmodels.py

```diff
@@ -118,6 +118,13 @@ class PageContent(models.Model):
     )
 
     objects = PageContentManager()
+    admin_manager = PageContentAdminManager()
```

Comment:
Could this new manager not inherit the existing `PageContentAdminManager` manager?

Reply:
Sorry, not sure what you mean here. I'd keep "user" and "admin" managers as separate as possible.

Reply:
It was more that the admin manager first inherits from the none admin manager, ensuring that any future changes to the standard manager are inherited. For sure the changes tot he admin manager should be separate. 

```
class PageContentAdminManager(PageContentManager):
    ...
```

Reply:
OK. Got it. Implemented it.